Track.h

```diff
@@ -587,6 +591,8 @@ inline int getScoreCalc(const unsigned int seedtype,
     if(pt<0.9f) score_ -= 0.2f*(Config::validHitBonus_)*nfoundhits;
     if(nfoundhits>8) score_ += (Config::validHitBonus_)*nfoundhits;
   }
+  */
+  if(score > 16000 && !Config::silent) std::cout << "WARNING!! Might be facing score overflow, which leads to discrepancies between STD and CE!" << std::endl;
```

Comment:
this will get messed up in multithreaded runs.
also, was this function in a vectorized part?
the cout is another badness.

Reply:
@slava77 do you have a suggestion for a better way to handle this?

Reply:
we have some print and dprint options.
@dan131riley @osschar I think that you know better than me.

Reply:
The Track class is an awkward place to report an error.  I guess I'd be in favor of adding a `static std::atomic<int>` error counter to the Track class.  Make the message a debug one with `dprint` and report any non-zero error counts at the end with the summary statistics.